sql/parser/builtins.go

```diff
@@ -116,6 +117,170 @@ var builtins = map[string][]builtin{
 		},
 	},
 
+	"concat_ws": {
```

Comment:
There doesn't appear to be any ordering to these functions currently. I think we should add some (perhaps in a follow-on PR to not disrupt the comments on this one too much). Either all alphabetical or perhaps groups (string functions, math functions, etc) and alphabetical within each group.